src/components/navigation/AdminNavMenu.jsx

```diff
@@ -29,9 +31,11 @@ const AdminNavMenu = () => {
     }
   };
 
+  const navigate = useNavigate();
+
   return (
     <>
-      {pathname !== '/' && (
+      {pathname !== '/auth' && (
```

Comment:
[no action required] usually i would ask you to make an array outside of the function and then do a `pathname in [list of no-render-navbar paths]` check but if the only path we're not rendering it for is `/auth` this time then it's fine